### ISSUE TITLE
Error when user-permissions plugin is disabled

### ISSUE THREAD

**Issue by jeffski** [PenguinOfWar/strapi-plugin-paper-trail#5]
In my Strapi CMS I have no requirement for users so I disabled the plugin by adding the following to the plugins.js file:

```javascript
'users-permissions': false,
```

That works fine and disables the User collection from the Content Manager.

When I installed and ran the CMS with paper-trail installed and enabled I get the following error which appears to be related to the user-permissions plugin.

```Error: Unknown target plugin::users-permissions.user
    at createJoinTable (/path/to/repo/node_modules/@strapi/database/lib/metadata/relations.js:409:11)
    at createOneToOne (/path/to/repo/node_modules/@strapi/database/lib/metadata/relations.js:45:7)
    at createRelation (/path/to/repo/node_modules/@strapi/database/lib/metadata/relations.js:344:14)
    at createMetadata (/path/to/repo/node_modules/@strapi/database/lib/metadata/index.js:76:11)
    at new Database (/path/to/repo/node_modules/@strapi/database/lib/index.js:19:21)
    at Database.init (/path/to/repo/node_modules/@strapi/database/lib/index.js:121:14)
    at Strapi.bootstrap (/path/to/repo/node_modules/@strapi/strapi/lib/Strapi.js:421:30)
    at Strapi.load (/path/to/repo/node_modules/@strapi/strapi/lib/Strapi.js:492:16)
    at async workerProcess (/path/to/repo/node_modules/@strapi/strapi/lib/commands/actions/develop/action.js:110:26)
Error: Error on attribute users_permissions_user in model trail(plugin::paper-trail.trail): Unknown target plugin::users-permissions.user
    at createMetadata (/path/to/repo/node_modules/@strapi/database/lib/metadata/index.js:83:15)
    at new Database (/path/to/repo/node_modules/@strapi/database/lib/index.js:19:21)
    at Database.init (/path/to/repo/node_modules/@strapi/database/lib/index.js:121:14)
    at Strapi.bootstrap (/path/to/repo/node_modules/@strapi/strapi/lib/Strapi.js:421:30)
    at Strapi.load (/path/to/repo/node_modules/@strapi/strapi/lib/Strapi.js:492:16)
    at async workerProcess (/path/to/repo/node_modules/@strapi/strapi/lib/commands/actions/develop/action.js:110:26)
```

When I re-enable the user-permissions plugin the paper-trial plugin starts and works fine.

It's something I can live with but would really like to not have to use the user plugin.

**Comment by PenguinOfWar** [comment]
Thanks for this, I’ll take a look and get a fix out.

**Comment by PenguinOfWar** [comment]
@jeffski This is now fixed in v0.3.0

https://github.com/PenguinOfWar/strapi-plugin-paper-trail/releases/tag/v0.3.0

Apologies for the time it took to deploy this fix; I had turned off my GitHub notifications accidentally! Let me know if you have any ongoing problems with the new version, and thanks again for the bug report.
